src/nn/var_store.rs

```diff
@@ -530,6 +530,20 @@ impl<'a> Path<'a> {
         self.f_var(name, dims, Init::KaimingUniform)
     }
 
+    /// Creats a new variable initialized randomly with an orthogonal matrix
```

Comment:
Could you fix the typo `Creats` -> `Creates`.

Reply:
Corrected. Thanks.